client/src/components/LoginModal.js

```diff
@@ -0,0 +1,114 @@
+// @flow
+
+import { ModalContext } from '@performant-software/shared-components';
+import React, { type Element } from 'react';
+import {
+  Button,
+  Divider,
+  Form,
+  Grid,
+  Header,
+  Icon,
+  Input,
+  Message,
+  Modal
+} from 'semantic-ui-react';
+import { useTranslation } from 'react-i18next';
+import './LoginModal.css';
```

Comment:
I think we're using CSS modules in Core Data. Can we do that here as well?

Reply:
On my local branch now I've renamed the CSS file, updated the import to `import styles from './LoginModal.module.css';`, updated the classnames to camel case, and updated my `className` props to e.g. `className={styles.loginModal}`. But the styles definitely aren't working. Do I need to configure something to get a new CSS module working?

Reply:
You'll need to add styles to the `Grid` component and any other components used in the selector(s):
```
<Grid
  className={cx(styles.ui, styles.grid)}
  ...
>
```
If you look at the DOM, any of the classes defined in the `.module` file are renamed from `ui grid` to a `ui_1234 grid_abcd` pattern.